private/performOptimizationFmincon.m

```diff
@@ -0,0 +1,67 @@
+function parameters = performOptimizationFmincon(parameters, negLogPost, iMS, par0, J_0, options)
```

Comment:
if you split this in computation and results assignment, you could reuse the code for profile computation. I suppose doing this until monday is asking for a lot, so please open a corresponding issue.